src/compiler/parser.ts

```diff
@@ -5698,9 +5700,10 @@ namespace ts {
             const hasJSDoc = hasPrecedingJSDocComment();
             parseExpected(SyntaxKind.WhileKeyword);
             const openParenPosition = scanner.getTokenPos();

```

Comment:
`scanner.getTokenPos` should also change to `getNodePos`, I'm pretty sure.

Reply:
bunp?

Reply:
(I'd still like scanner.getTokenPos changed to getNodePos)

Reply:
Updated. The latest commit has the baseline changes. I'm suspicious of the jsx baseline changes... I think we had to edit around a separate jsx change when we did the original revert.